stdlib/number.gr

```diff
@@ -660,3 +660,89 @@ provide let toRadians = degrees => degrees * (pi / 180)
  * @since v0.5.4
  */
 provide let toDegrees = radians => radians * (180 / pi)
+
+/**
+ * Clamps a number to a given range.
```

Comment:
We don't use the name of the function to describe the function. Can you reword?

Reply:
I also believe the description needs to mention if the range is treated as inclusive or exclusive. It seems to be in the param but not in the description.